packages/vinxi/lib/index.js

```diff
@@ -3,3 +3,5 @@ export * from "./app.js";
 /** @typedef {import('vite').Plugin} Plugin */
 export { resolve } from "./resolve.js";
 export * from "./router-modes.js";
+
+/** @typedef {import('nitropack').NitroConfig} NitroConfig */
```

Comment:
Actually I might not need this. Could just do 

import { AppOptions } from "vinxi";
server?: AppOptions['server'],

Reply:
Yeah I think so. Or we can export AppServerOptions. But I think using AppOptions is fine too. Minimal API surface area is better to maintain